roles/ps_agent/tasks/main.yml

```diff
@@ -1,43 +1,39 @@
 - name: Install gpg
-  become: yes
```

Comment:
Dropping root seems like outside the scope of this pr

Reply:
So the playbook has `become: true` (https://github.com/Paperspace/machine-metrics/blob/main/metrics.yml#L2). All these are redundant in this context, since this role is not intended to be called separately from the playbook.